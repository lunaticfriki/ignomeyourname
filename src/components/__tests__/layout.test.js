import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../layout'

describe('Layout component suite', () => {
  test('Layout snapshot', () => {
    const component = renderer.create(<Layout />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
