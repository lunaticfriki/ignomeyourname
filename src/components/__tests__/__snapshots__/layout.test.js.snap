// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component suite Layout snapshot 1`] = `
<div
  className="sc-AxiKw cZthxE"
>
  <div
    className="sc-AxjAm RhMFK"
  >
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "#d62b9a",
        }
      }
    >
      <h1>
        I GNOME YOUR NAME!
      </h1>
    </a>
    <h2>
      Search your favorite gnome from Brastlewark!
    </h2>
    <nav>
      <a
        aria-current="page"
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "#d62b9a",
          }
        }
      >
        <li>
          Searcher
        </li>
      </a>
      <a
        href="/gallery"
        onClick={[Function]}
        style={
          Object {
            "color": "#fff",
          }
        }
      >
        <li>
          Gallery
        </li>
      </a>
      <a
        href="/about"
        onClick={[Function]}
        style={
          Object {
            "color": "#fff",
          }
        }
      >
        <li>
          About me
        </li>
      </a>
    </nav>
  </div>
  <div
    className="sc-AxirZ cmQgFr"
  >
    <i>
      oscar ventura - gnome searcher - 2020
    </i>
  </div>
</div>
`;
